tests/data/manifest/manifest0.yaml

```diff
@@ -16,7 +16,6 @@ registry:
         repo: nixpkgs-flox
```

Comment:
Remove `nixpkgs-flox` and `floxpkgs` input entirely from any test data files.

This is expected to change the number of results in some tests, so feel free to update those test cases as well.

Reply:
I'll do this in next PR